faqs.md

```diff
@@ -0,0 +1,155 @@
+<img src="./images/logos/colour text.png" alt="GMN logo, a coloured GMN logo with transparent background." width="400"  align="right" />
```

Comment:
```suggestion
```

Reply:
The logo would be likely in the template for the pages. It does not need to be added in each Mardkown file